server/test/testdatasets/flyingSites.json

```diff
@@ -56,9 +56,10 @@
   },
   {
     "name": "Gangelsberg",
-    "heightDifference": "300",
+    "heightDifference": "170",
     "image": "",
     "region": "Pfalz",
+    "country": "RP",
```

Comment:
eher state, oder?

Reply:
"state" ist schon in dem Model belegt im Sinne von "Status" (active, inactive, proposal).

Reply:
Das macht Country aber ja nicht richtiger.
Wir können ja country hinzufügen und stateCode (BL Abkürzungen => State Code). Dann kann man da auch zwischen Bundesländern und Luxemburg unterscheiden.